# ctla4_mtec

This repo contains scripts used to analyze bulk RNA-seq, single-cell RNA-seq and microarray data for the manuscript
Michelson et al, "Ctla-4 on thymic epithelial cells complements Aire for T-cell central tolerance" (2022).

These analyses made use of:

Bulk RNA-seq data from the following papers:
Sansom et al, "Population and single-cell genomics reveal the Aire dependency, relief from Polycomb silencing, and distribution of self-antigen expression in thymic epithelia" Genome Res (2014).
Bansal et al, "Aire regulates chromatin looping by evicting CTCF from domain boundaries and favoring accumulation of cohesin on superenhancers" PNAS (2021).
Meredith et al, "Aire controls gene expression in the thymic epithelium with ordered stochasticity" Nat Immunol (2015).
Michelson et al, "Thymic epithelial cells co-opt lineage-defining transcription factors to eliminate autoreactive T cells" Cell (2022).

scRNA-seq data from the following papers:
Michelson et al, "Thymic epithelial cells co-opt lineage-defining transcription factors to eliminate autoreactive T cells" Cell (2022).

Microarray data from the following papers:
Venanzi et al, "The variable immunological self: genetic variation and nongenetic noise in Aire-regulated transcription" PNAS (2008).